troposphere/static/js/components/projects/resources/link/details/sections/ExternalLinkActions.react.js

```diff
@@ -16,11 +16,11 @@ define(function (require) {
 
     onDelete: function () {
       var project = this.props.project,
-        link = this.props.link;
+        external_link = this.props.link;
 
       modals.ExternalLinkModals.destroy({
-        link: this.props.link,
-        project: this.props.project
+        project,
+        external_link
```

Comment:
If `onDelete` is broken - have we verified that `handleReport` is still working? I'd be it doesn't since it's passing along `link` and not `external_link`.